src/Analyzers/CSharp/CodeFixes/UsePrimaryConstructor/CSharpUsePrimaryConstructorCodeFixProvider.cs

```diff
@@ -181,8 +181,8 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
                     currentTypeDeclaration, constructorDeclaration, constructor, properties, removedMembers);
 
                 return currentTypeDeclaration
-                    .WithLeadingTrivia(finalTrivia)
                     .WithAttributeLists(finalAttributeLists)
+                    .WithLeadingTrivia(finalTrivia)
```

Comment:
oops.  by adding the attributes after the trivia, we were blowing away the new trivia we'd just set with the original trivia on the node.